it/mail/chrome/messenger/addressbook/ldapAutoCompErrs.properties

```diff
@@ -91,48 +91,48 @@ errorAlertFormat=Codice di errore %1$S: %2$S\n\n %3$S
 ## @loc none
-10085=Riprovare più tardi.
+10085=Riprova più tardi.
 
 ## @name FILTER_ERROR_HINT
 ## @loc none
-10087=Verificare che il filtro di ricerca sia corretto quindi riprovare o contattare l'Amministratore di Sistema. Per verificare la correttezza del filtro dal menu Modifica scegliere Preferenze, quindi Posta e gruppi di discussione ed infine Indirizzamento. Fare clic su Modifica Directory e selezionare il server LDAP che si sta usando. Infine fare clic su Modifica e quindi su Avanzate per mostrare il filtro di ricerca.
+10087=Verifica che il filtro di ricerca sia corretto e poi riprova oppure contatta il tuo Amministratore di sistema. Per verificare che il filtro di ricerca sia corretto, scegli Preferenze dal menu Modifica, poi E-mail e Newsgroup e poi Indirizzamento. Clicca su Modifica directory e seleziona il server LDAP che stai utilizzando. Clicca su Modifica e poi clicca su Avanzate per mostrare il Filtro di ricerca.
 
 ## @name NO_MEMORY_HINT
 ## @loc none
-10090=Chiudere qualche altra applicazione e/o finestra e riprovare.
+10090=Chiudi altre finestre e/o applicazioni e riprova.
 
 ## @name CONNECT_ERROR_HINT
 ## @loc none
-10091=Verificare che il nome dell'host remoto e la porta siano corretti quindi riprovare o contattare l'Amministratore di Sistema. Per verificare la correttezza del nome dell'host remoto e della porta dal menu Modifica scegliere Preferenze, quindi Posta e gruppi di discussione ed infine Indirizzamento. Fare clic su Modifica Directory e selezionare il server LDAP che si sta usando. Infine fare clic su Modifica per visualizzare il nome dell'host remoto e Avanzate per visualizzare la porta.
+10091=Verifica che il nome dell'host e il numero della porta siano corretti e poi riprova oppure contatta il tuo Amministratore di sistema. Per verificare che il nome dell'host e il numero della porta siano corretti, scegli Preferenze dal menu Modifica, poi E-mail e Newsgroup e poi Indirizzamento. Clicca su Modifica directory e seleziona il server LDAP che stai utilizzando. Clicca su Modifica per visualizzare il nome dell'host. Clicca su Avanzate per visualizzare il numero della porta.
 
 ## @name HOST_NOT_FOUND_HINT
 ## @loc none
-15000=Controllare che il nome dell'host sia correte, quindi tentare di nuovo o contattare l'amministrazione di sistema. Per verificare che il nome dell'host (hostname) sia corretto, dal menu' modifica , scegliere preferenza, quindi scegliere Posta e gruppi di discussione, quindi scegliere indirizzamento. Clic su Edita cartelle, e selezionare LDAP come server usato. Clic su Modifica per mostrare il nome dell'host.
+15000=Verifica che il nome dell'host sia corretto, poi riprova oppure contatta il tuo Amministrazione di sistema. Per verificare che il nome dell'host sia corretto, scegli Preferenze dal menu Modifica, poi scegli E-mail e Newsgroup e poi Indirizzamento. Clicca su Modifica directory e seleziona il server LDAP che stai utilizzando. Clicca su Modifica per mostrare il nome dell'host.
```

Comment:
CORRECTIONS NEEDED. It should be as follows: 

 +15000=Verifica che il nome dell'host sia corretto poi riprova, oppure contatta l'Amministratore di sistema. Per verificare che il nome dell'host sia corretto, scegli Preferenze dal menu Modifica, poi scegli E-mail e Newsgroup e poi Indirizzamento. Clicca su Modifica directory e seleziona il server LDAP che stai utilizzando. Clicca su Modifica per mostrare il nome dell'host.